Dockerfile

```diff
@@ -20,18 +25,9 @@ ENV COMPOSER_ALLOW_SUPERUSER 1
 
 COPY php-cli.ini /usr/local/etc/php/
 COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
```

Comment:
I reckon we no longer need to copy composer into the final satis image then anymore, don't you think so too?

Reply:
indeed. it's not needed there. removing